subsys/net/lib/sntp/sntp_pkt.h

```diff
@@ -9,21 +9,21 @@
 
 #include <zephyr/types.h>
 
-#define LI_MASK   0xC0
-#define VN_MASK   0x38
-#define MODE_MASK 0x07
+#define Z_SNTP_LI_MASK   0xC0
+#define Z_SNTP_VN_MASK   0x38
+#define Z_SNTP_MODE_MASK 0x07
 
-#define LI_SHIFT   6
-#define VN_SHIFT   3
-#define MODE_SHIFT 0
+#define Z_SNTP_LI_SHIFT   6
+#define Z_SNTP_VN_SHIFT   3
+#define Z_SNTP_MODE_SHIFT 0
 
-#define LVM_GET_LI(x)    ((x & LI_MASK) >> LI_SHIFT)
-#define LVM_GET_VN(x)    ((x & VN_MASK) >> VN_SHIFT)
-#define LVM_GET_MODE(x)  ((x & MODE_MASK) >> MODE_SHIFT)
+#define LVM_GET_LI(x)    ((x & Z_SNTP_LI_MASK) >> Z_SNTP_LI_SHIFT)
```

Comment:
And these LVM_ macros will probably need the SNTP_ prefix as well.

Reply:
> And these LVM_ macros will probably need the SNTP_ prefix as well.

Quoting @jukkar, I'd consider it "superfluous atm."

Or otherwise, I would need to figure what that "LVM_" means, and whether it should be there in the first place. That's quite faraway from the original task which triggered this work, https://github.com/zephyrproject-rtos/zephyr/issues/24730

Reply:
no the Z_ is superfluous, not the SNTP_ 

note that I always prefer to use domain prefix on macros (either driver name if it's a driver, etc...) instead of no prefix. So this change is a good one imo

Reply:
> note that I always prefer ...

And I prefer a namespace prefix for the entire project, so what?

> So this change is a good one imo

Done. I dropped "LVM_", as it's not a standard SNTP term, there it was only used to refer to "lvm" field of the defined `struct`.